modules/nf-core/sentieon/tnhaplotyper2/tests/main.nf.test

```diff
@@ -0,0 +1,378 @@
+nextflow_process {
+
+    name "Test Process SENTIEON_TNHAPLOTYPER2"
+    config "./nextflow.config"
+    script "../main.nf"
+    process "SENTIEON_TNHAPLOTYPER2"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "picard"
+    tag "picard/addorreplacereadgroups"
+    tag "sentieon"
+    tag "sentieon/tnhaplotyper2"
+    config './nextflow.config'
+
+    setup {
+        run("PICARD_ADDORREPLACEREADGROUPS", alias: "NORMAL_ADDORREPLACEREADGROUPS") {
+            script "../../../picard/addorreplacereadgroups/main.nf"
+            config './nextflow.config'
+            process {
+                """
+                input[0] = [[id:'test'],// meta map
+                    [file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.paired_end.recalibrated.sorted.bam',checkIfExists:true)]
+                    ]
+                input[1] = [[],[]]
+                input[2] = [[],[]]
+                """
+            }
+        }
+
+        run("PICARD_ADDORREPLACEREADGROUPS", alias: "TUMOUR_ADDORREPLACEREADGROUPS") {
+            script "../../../picard/addorreplacereadgroups/main.nf"
+            config './nextflow.config'
+
+            process {
+                """
+                input[0] = [[id:'test'],// meta map
+                    [file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test2.paired_end.recalibrated.sorted.bam',checkIfExists:true)]
+                    ]
+                input[1] = [[],[]]
+                input[2] = [[],[]]
+                """
+            }
+        }
+    }
+
+    test("no pon") {
+
+        when {
+            params {
+                module_args  = ''
+                module_args2 = '--tumor_sample tumour --normal_sample normal'
+                module_args3 = ''
+                module_args4 = ''
+            }
+            process {
+                """
+                input[0] = NORMAL_ADDORREPLACEREADGROUPS.out.bam
+                    .join( NORMAL_ADDORREPLACEREADGROUPS.out.bai)
+                    .join( TUMOUR_ADDORREPLACEREADGROUPS.out.bam)
+                    .join( TUMOUR_ADDORREPLACEREADGROUPS.out.bai)
+                    .map{
+                    meta,normal_bam,normal_bai,tumour_bam,tumour_bai->
+                    [[id:'test'],[normal_bam,tumour_bam],[normal_bai,tumour_bai],[]]
+                    }
+                input[1] = [[:],file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.dict',checkIfExists:true)]
```

Comment:
I don't know if it is something that we want in the main.nf.test, but could you add some spaces after a comma ?